app/Console/Commands/VATUSAEventsUpdate.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Event;
+use Carbon\Carbon;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+
+class VATUSAEventsUpdate extends Command {
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'VATUSAEvents:Update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Fetches events from VATUSA and syncs id_topics in the events table.';
+
+    protected $apiEndpoint = "https://api.vatusa.net/v2/public/events/";
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct() {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle() {
+        $vatusa_events = [];
+        $vatusa_json = $this->getEvents();
+        if (!is_array($vatusa_events)) {
+            return null;
+        }
```

Comment:
This will never be false because `$vatusa_events` is hard coded as an array

Reply:
Also to early return a function, you should just be able to use `return`. This won't return anything so you're not returning null into space.

Reply:
This conditional provide protection in case $this->getEvents() returns a non-array. From the php documentation for  json_decode(), "null is returned if the json cannot be decoded or if the encoded data is deeper than the nesting limit." Since this can happen if the VATUSA API does not issue a response, it is important that we protect the execution sequence so it does not fail.

I will fix the return - thank you for pointing that out.

Reply:
Ohh gotcha this must be a typo - you'd want it to be `!is_array($vatusa_json)`

Reply:
Right you are! LOL - updated and fixed